### DISCUSSION TITLE
nvm

### DISCUSSION THREAD

**Discussion by ckdlskgkdrhd** [naverz/zepeto-studio-global#70] [General]
I have my character run down to a steep fall,
 
![캡처dsdsdsds](https://user-images.githubusercontent.com/110361648/188366572-a1c6c25d-353a-4fc9-8ed9-3911326c7080.PNG)
![zzzzzssdd](https://user-images.githubusercontent.com/110361648/188366606-0c6063c8-32f0-48ca-ac42-0aaeb1a419d7.PNG)

teleport line wouldn't be executed.

but when I jump to fall down
it works as intended.

**edit: had to update zepeto.world to 1.6**

**Comment by naverz-mkao** [comment]
Hi,

Could you please provide your code for this so we can take a look?

Thanks!

**Reply by ckdlskgkdrhd** [reply]
nah all good. 
I just had to update my zepeto.world package from 1.3 to 1.6
